backends/bmv2/common/controlFlowGraph.cpp

```diff
@@ -166,7 +165,7 @@ bool CFG::checkImplementable() const {
 namespace {
 class CFGBuilder : public Inspector {
     CFG *cfg;
-    /// predecessors of current CFG node
```

Comment:
This is not correct. Members in header files also will have triple slashes. We only use double slashes for "internal" comments within functions, for example.

Reply:
Please fix this for all instances.

Reply:
/// Create a header type in json.
 /// @param name header name
 /// @param type header type
 /// @param max_length  maximum length for a header with varbit fields;
 ///                    if 0 header does not contain varbit fields
 /// @param fields a JsonArray for the fields in the header

Are you referring to a gap in these comments?

Reply:
You can format Github comments using triple '`', it will make your comment more readable: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Yes, in all the comments where there is a newline between the comments and the actual function.